### PR TITLE
messaging: windows special pages

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1208228079594252/1208768638191551/f

## Description

Added CEF Support for Windows on special pages.

## Testing Steps

- tested for real by @q71114 as part of the integration 

## Checklist

*Please tick all that apply:*

- [ ] I have tested this change locally
- [x] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [x] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Review by q71114 — DISMISSED** [review]
Approving this as the corresponding windows PR is approved now: https://github.com/duckduckgo/windows-browser/pull/2627

**Review by q71114 — DISMISSED** [review]
Integration tested in https://github.com/duckduckgo/windows-browser/pull/2627

**Comment by shakyShane** [comment]
Thanks @q71114  I updated the tests, once passing I'll get this merged :)

File: messaging/lib/test-utils.mjs
Comment (shakyShane) [inline messaging/lib/test-utils.mjs:1]: This is step 1, we'll be able to remove the other one soon hopefully!

**Pull request merged**